src/SignupRouter.js

```diff
@@ -45,6 +45,12 @@ const fullScreenContainer = {
   flexDirection: 'column',
 }
 
+let heightAndroidDevice = 0
+
+if (!heightAndroidDevice && isAndroid) {
+  heightAndroidDevice = getScreenHeight()
```

Comment:
this should be a variable saved in orientation. originalScreenHeight. export getOriginalScreenHeight

Reply:
done to https://github.com/GoodDollar/GoodDAPP/pull/1064/commits/b3b782996b82c47d8d62aef7bac716f7ab200f03